code-build.js

```diff
@@ -16,14 +16,16 @@ module.exports = {
   logName
 };
 
-function runBuild() {
+async function runBuild() {
   // get a codeBuild instance from the SDK
   const sdk = buildSdk();
 
   // Get input options for startBuild
   const params = inputs2Parameters(githubInputs());
 
-  return build(sdk, params);
+  console.log("*****STARTING CODEBUILD*****");
```

Comment:
I would put this in the top level `run` method.
The waiting to actually start I don't think offers too much value.
But if you really want to have this here, you need to return the `build`.
see (https://github.com/aws-actions/aws-codebuild-run-build/blob/master/index.js)

Reply:
Ah; good point. Will do.